doc/workshops/financial.rst

```diff
@@ -2,4 +2,1030 @@
+   
+.. note::
+   
+   Spyder now offers :ref:`standalone_installers_ref` for Windows and macOS, making it easier to get up and running with the application without having to download Anaconda or manually install it in your existing environment.
+
+You can also install the virtual environment easily using the environment file that we share with you. Just run the following command in the terminal (you must have the environment file in the current directory):
+
+.. code-block:: bash
+   
+   $ conda env create -f workshop-1-environment.yml
+
+You can now enter the newly created virtual environment in this way:
+
+.. code-block:: bash
+   
+   $ conda activate spyder-for-finance
+
+   
+Download the datasets
+~~~~~~~~~~~~~~~~~~~~~
+   
+Although during the workshop we will explain how to use some APIs to download up-to-date data, you can also download the datasets in csv format from these links. [ENTER MISSING LINKS]
+
+To follow this workshop you do not need to create a new directory. However, if you have downloaded the data and want to use it instead of the APIs, you must set the directory that has the downloaded data as the working directory. In order to do this, check that the working directory is correct. You should see in the upper right corner the path to the directory where you have the downloaded data. Something like this:
+
+.. image:: images/workshop-1/working-directory.png
+   :alt: Spyder's Working Directory plugin showing the working directory downloaded-data-dir
+
+
+Set up the virtual environment in Spyder
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's check that the virtual environment we created is enabled in Spyder. Go to Preferences > Python interpreter, and under "Use the following Python interpreter" add the path to your virtual environment. You should see something like this:
+
+.. image:: images/workshop-1/set-up-environment.png
+   :alt: Set up environment in Spyder's Preferences
+
+Now, you have everything ready to proceed with the workshop.
+   
+   
+   
+=====================
+Obtain financial data
+=====================
+   
+When it comes to finance, being up to date is very important. So we are going to use a Python library that allows us to get updated historical Stock Market records from `Yahoo! Finance`_ API. In this way, we will be able to download data in the period of time we are interested in analyzing.
+
+.. _`Yahoo! Finance`: https://finance.yahoo.com
+
+To get started, import the libraries.
+
+.. important:: 
+
+   Write and run all code for this workshop in the "IPython Console" at the bottom left of Spyder.
+
+.. code-block:: python
+      
+   import numpy as np
+   import pandas as pd
+   from pylab import mpl
+   from scipy.optimize import minimize
+   import matplotlib.pyplot as plt
+   import pprint
+   import yfinance as yf
+
+The first group of imports is for basic operations. The second imports the libraries that we will use to download financial data.
```

Comment:
I don't know if it's useful to mention two groups, if they are actually merged into one in the code. Maybe leave a blank like between groups so this is clearer? Otherwise you can just mention the libraries in each group by name.